folium/utilities.py

```diff
@@ -365,7 +365,8 @@ def temp_html_filepath(data: str) -> Iterator[str]:
         yield filepath
     finally:
         if os.path.isfile(filepath):
-            os.remove(filepath)
+            pass
+            # os.remove(filepath)
```

Comment:
Guess this is an artefact from testing? Should be restored.

```suggestion
            os.remove(filepath)
```